src/execution_plan/ops/op_unwind.h

```diff
@@ -14,18 +14,22 @@
 
 typedef struct {
 	OpBase op;
-	SIValue list;         // List which the unwind operation is performed on.
-	SIValue from;         // From range
-	SIValue to;           // To range
-	SIValue step;         // Step range
-	SIValue current;      // current range
-	bool is_range;
-	AR_ExpNode *exp;      // Arithmetic expression (evaluated as an SIArray).
-	uint listIdx;         // Current list index.
-	int unwindRecIdx;     // Update record at this index.
-	Record currentRecord; // record to clone and add a value extracted from the list.
+	SIValue list;         // list which the unwind operation is performed on
+	int64_t from;         // beginning of range
+	int64_t to;           // end of range
+	int64_t step;         // step size
+	int64_t current;      // current value in range
+	bool is_range;        // op is using range otherwise list
```

Comment:
What do you think about using union here?
```c
UNION {
struct {
int64_t from;         // beginning of range
 	int64_t to;           // end of range
 	int64_t step;         // step size
 	int64_t current;      // current value in range
};
struct {
 	SIValue list;
 	uint listIdx;         // current list index
}
```

This way there's a clear distinction between the two scenarios